spec/recipes/solaris2_spec.rb

```diff
@@ -13,7 +13,7 @@
     expect(chef_run).to install_package('gnu-coreutils')
     expect(chef_run).to install_package('flex')
     expect(chef_run).to install_package('gcc').with_version('4.8.2')
-    expect(chef_run).to install_package('gcc-3')
+    expect(chef_run).to install_package('gcc-c').with_version('4.8.2')
```

Comment:
should there also be a unit test for `gcc-c++`?

Reply:
Added